src/components/select/select.js

```diff
@@ -1136,11 +1142,11 @@ function SelectProvider($$interimElementProvider) {
            */
           function mouseOnScrollbar() {
             var clickOnScrollbar = false;
-            if (ev && (ev.currentTarget.children.length > 0)) {
+            if (mouseDownXPosition != null && ev && (ev.currentTarget.children.length > 0)) {
```

Comment:
Completely understandable. Another question btw: Would it be a better way, or more angular way, to make a container element with a content child to have a clear distinction between mouseevents? Container mouseevent would then always originate from the scrollbar and content mouseevent always from a mdOption.